wp/lib/bap_wp/src/run_parameters.ml

```diff
@@ -56,6 +58,13 @@ type t = {
   ext_solver_path : string option
 }
 
+type loop_invariant = {
+  address : string;
```

Comment:
is this a hex address, or a tid?

Reply:
Hex address in the form `(address 0x12:64u)`. I couldn't figure out how to use the tid labels from the KB.

Reply:
Can we infer the `:64u`? I suspect users may not want to type that.